File: week-2/lecture/scores.c
#include <cs50.h>
#include <stdio.h>

float average(int length, int array[]);

int N = 3;

int main(void)
{
  int scores[N];

  for (int i = 0; i < N; i++)
  {
    scores[i] = get_int("Score: ");
  }

  printf("Average: %f\n", average(N, scores));

}

float average(int length, int array[])
{
  int total = 0;

  for (int i = 0; i < length; i++)
  {
    total += array[i];
  }

  return total / (float) (length);
}

File: week-6/lecture/scores.py
scores = []


def main():
    # getting score
    add_score(3)

    # calculating average
    average = sum(scores) / len(scores)

    # printing
    print(f"average is {average}")


def add_score(n):
    for _ in range(n):
        while True:
            try:
                score = int(input("Score: "))
                if score > 0:
                    scores.append(score)
                    break
            except ValueError:
                print("Not an integer!")


main()
